process_report/process_report.py

```diff
@@ -184,6 +185,12 @@ def main():
         default="pi_invoices",
         help="Name of output folder containing pi-specific invoice csvs",
     )
+    parser.add_argument(
+        "--prepay-groups-output-folder",
```

Comment:
Let's not make this configurable but provide default locations for invoices moving forward. Name should be hardcoded in the invoice.